src/main/java/frc/robot/Constants.java

```diff
@@ -110,8 +112,42 @@ public static final class NeoMotorConstants {
 
   public static final class OperatorConstants {
     public static final int kDriverControllerPort = 0;
+    public static final int leftJoystickAxis = 1;
+    public static final double joystickDeadband = 0.08;
+    public static int kOperatorControllerPort;
+  }
+
+  public static final class MotorConstants {
+    public static final int motorID = 6;
+    public static final double maxSpeed = 0.5;
+    public static final double autoSpeed = 0.2;
+    public static final double autoTimeout = 5;
+  }
+
+  public static final class EncoderConstants {
+    public static final double wheelRadius = 3;
+    public static final double gearRatio = 8.46;
+    public static final double ticksPerRevolution = 4096;
+    public static final double encoderCountsToMeters = Units.inchesToMeters((2 * Math.PI * wheelRadius) / gearRatio);
+  }
+
+  public static final class PIDConstants {
+    public static final double pos_kp = 0.1;
+    public static final double pos_ki = 0.1;
+    public static final double pos_kd = 0.1;
+  }
+
+  public static final class ShooterConstants {
+    public static final int kTopShooterMotorPort = 0;
+    public static final int kBottomShooterMotorPort = 0;
+
+    public static final double kPShooterController = 0.000002;
+    public static final double kIShooterController = 0.0;
+    public static final double kDShooterController = 0.0008;
+    public static final double kShooterVelocityFF = 0.0001695; // modify based on error on actual shooter testing
   }
```

Comment:
If all of these are for the shooter, can they be put in `ShooterConstants` so we have everything isolated between subsystems?

Please add the `k` in front for convention. Also, it's good to be specific about what the values mean, `gearRatio` or  `motorID` don't mean much. It something like `kGearReduction` and `kTopMotorCanId` remove any ambiguity to what the values mean.